sources/postgres/toddl_test.go

```diff
@@ -106,14 +106,15 @@ func TestToExperimentalSpannerType(t *testing.T) {
 	name := "test"
 	srcSchema := schema.Table{
 		Name:     name,
-		ColNames: []string{"a", "b", "c", "d", "e", "f"},
+		ColNames: []string{"a", "b", "c", "d", "e", "f", "g"},
 		ColDefs: map[string]schema.Column{
```

Comment:
This test function is specific to capturing unsupported types for PG Spanner - is TestProcessPgDump a better place to capture new types that HB now supports? You would be able to add test for both schema and data conversion there.